### PR TITLE
Update demo app README with troubleshooting steps

### PR DESCRIPTION
**Pull request by prameshj**
Updated the readme with screenshots and troubleshooting steps.

@sam-gc @renkelvin

### CODE REVIEW

**Review by renkelvin — APPROVED** [review]

File: packages/auth/demo/README.md [hunk 1]
```diff
@@ -74,3 +89,43 @@ The demo page by default runs against the actual Auth Backend. To run against th
 yarn run demo:emulator
 ```
 
+## Troubleshooting
+
+### Errors about dependency not being installed, example `lerna: command not found`
+  
+  Ensure that you run `yarn` to install dependencies.
+
```
Comment (Xiaoshouzi-gh) [inline packages/auth/demo/README.md:97]: I also run into a similar problem when run `firebase use --add`.
We can also add another troubleshooting here -
`Failed to list Firebase projects.`
Logout, re-login
firebase logout && firebase login

File: packages/auth/demo/README.md [hunk 2]
```diff
@@ -58,6 +59,20 @@ yarn run demo
...
+If not, you will see an “auth/operation-not-allowed” error message. 
+
+You can check the enabled providers on the firebase console.
+
+![image](https://user-images.githubusercontent.com/35932340/153662750-c0faf417-07b4-4f0e-93ab-5e0b82e3c793.png)
+
+
 ## Running against Auth Emulator
 
 The demo page by default runs against the actual Auth Backend. To run against the Auth Emulator with mocked endpoints, do the following:
```
Comment (Xiaoshouzi-gh) [inline packages/auth/demo/README.md:78]: Can we add a pre-requisite for this step - 
You need to have Java install.

**Review by Xiaoshouzi-gh — CHANGES_REQUESTED** [review]
Content in the copy config section needs more clarification.
From ```Then copy and paste the Web snippet config found in the console (either by clicking "Add Firebase to
your web app" button in your Project overview, or clicking the "Web setup" button in the Auth page)
in the `config.js` file.```

to
```
Redirect to Firebase console, click on Project Overview from the side bar.  Click on Add app then choose `Web`.  Fill in App nickname and leave the `Also set up Fiebase Hosting for this app` unchecked for now.  Click on Register app.  In the Add Firebase SDK section, copy over the firebaseconfig object to your `config.js` file config object.
```
my guess is - before register app, it will show 0 app in the project.  We might not able to find this config anywhere.

File: packages/auth/demo/README.md (hunk 2 above)
Comment (prameshj) [reply packages/auth/demo/README.md:79]: Java prereq is specified in the top level readme. I linked to it in this doc - https://github.com/firebase/firebase-js-sdk#prerequisites

**Comment by prameshj** [comment]
> Content in the copy config section needs more clarification. From `` Then copy and paste the Web snippet config found in the console (either by clicking "Add Firebase to your web app" button in your Project overview, or clicking the "Web setup" button in the Auth page) in the `config.js` file. ``
> 
> to
> 
> ```
> Redirect to Firebase console, click on Project Overview from the side bar.  Click on Add app then choose `Web`.  Fill in App nickname and leave the `Also set up Fiebase Hosting for this app` unchecked for now.  Click on Register app.  In the Add Firebase SDK section, copy over the firebaseconfig object to your `config.js` file config object.
> ```
> 
> my guess is - before register app, it will show 0 app in the project. We might not able to find this config anywhere.

This doc specifies 

> Content in the copy config section needs more clarification. From `` Then copy and paste the Web snippet config found in the console (either by clicking "Add Firebase to your web app" button in your Project overview, or clicking the "Web setup" button in the Auth page) in the `config.js` file. ``
> 
> to
> 
> ```
> Redirect to Firebase console, click on Project Overview from the side bar.  Click on Add app then choose `Web`.  Fill in App nickname and leave the `Also set up Fiebase Hosting for this app` unchecked for now.  Click on Register app.  In the Add Firebase SDK section, copy over the firebaseconfig object to your `config.js` file config object.
> ```
> 
> my guess is - before register app, it will show 0 app in the project. We might not able to find this config anywhere.

This doc specifies `You need to have created a Firebase Project in the
[Firebase Console](https://firebase.google.com/console/) as well as configured a web app.` as a pre-req, so the user should have one app here. Is is ok to skip this?

**Review by sam-gc — APPROVED** [review]

**Pull request merged**